lib/shopify_transporter/exporters/magento/order_exporter.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module ShopifyTransporter
+  module Exporters
+    module Magento
+      class OrderExporter
+        def initialize(store_id: nil, client: nil)
+          @client = client
+          @store_id = store_id
+        end
+
+        def export
+          $stderr.puts "Starting export..."
```

Comment:
Replace `$stderr` with `puts`, we are are not logging bad things and can simply log to stdout.